exams/utils.py

```diff
@@ -59,3 +67,53 @@ def authorize_for_exam(mmtrack, course_run):
                 mmtrack.user.username,
                 course_run.edx_course_key
             )
+        else:
+            log.info(
+                'Unable to authorize user "%s" for exam, course id "%s"',
+                mmtrack.user.username,
+                course_run.edx_course_key
+            )
+
+
+def bulk_authorize_for_exam(program_id=None, username=None, course_key=None):
+    """
+    Authorize user(s) for exam(s).
+
+    Args:
+        program_id(str): Program id (optional) if you want authorization on spcific program exams
+        username(str): User name (optional) whom you want to authorize for exam(s)
+        course_key(str): Edx course key  (optional) where you want to authorize user(s)
+    """
+    programs = Program.objects.filter(exam_series_code__isnull=False, live=True).filter(~Q(exam_series_code=""))
```

Comment:
I would suggest refactoring these filters a little so it's a little easier to read:

```python
Program.objects.filter(live=True).exclude(exam_series_code__isnull=True).exclude(exam_series_code__exact="")
```